coding-concepts.js

```diff
@@ -29,23 +29,23 @@ var languages = ["JavaScript", "Ruby", "Python", "C++"]
 var index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: Answer correct because there is a zero index when locating the place of a character...so the count starts at 0. Each language is counted as a single location because it is in an array.
 
 
 // --------------------4) What will this log?
 
 var weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: SATURDAY, SUNDAY
+// b) Verify and explain: "weekendDays.toUpperCase is not a function." (I copied the stuff in quotations from the console. All these other words are mine.) I am assuming there is some extra steps needed to capitalize an array. Join function to make a string and then use .toUpperCase(). My head hurts too much from trying to reverse that other array.
```

Comment:
Correct! Even with your head hurting you figured it out! Yes, .toUpperCase() does not work on arrays it only works on strings.